src/server/auth-passport.js

```diff
@@ -119,9 +120,8 @@ export function setupLocalAuthPassport() {
 
   return {
     loginCallback: [
-      passport.authenticate('local', { failureRedirect: '/login' }),
+      passport.authenticate('local'),
```

Comment:
Can you explain why we remove the failureRedirect path? Not in code comments, necessarily, but to me?

If it's because we 'cannot fail' above because we either create a user or login, then that won't be the case after testing for the conditions necessary for account creation, and then we'll need it again.

Reply:
I removed it because the server response was coming back with a `302 Found` or `500 Internal Server Error` status code (depending on how verification is handled here: https://github.com/MoveOnOrg/Spoke/pull/1059/files#diff-5f814da17f81ded4f5bf5d984739c0b0R92). Without redirecting, Passport by default will respond with a `401 Unauthorized` status if authentication fails, which to me made more sense.

Reply:
Hmm. I think we do want it to redirect to login if they are unauthorized. If that's not the way to do it then I think we need to figure out how. Maybe I setup the options incorrectly?

Reply:
Hey @schuyler1d, from what I understand, since we are initiating the `POST` request via JS, the browser won't actually redirect, even if the response from the server comes with a redirect request. So right now, the user won't actually leave the login page until their authentication request is successful.

Is that your concern, or are you thinking of something else? If you'd like, we can chat about it so I can better understand where you are coming from.

Thanks!